kong/plugins/acme/schema.lua

```diff
@@ -42,6 +42,7 @@ local LEGACY_SCHEMA_TRANSLATIONS = {
   { auth = {
     type = "string",
     len_min = 0,
+    translate_backwards = {'password'},
```

Comment:
HI @nowNick I'm just curious about this PR, I noticed you add `password` here to output it to Konnect users as you said in KAG description. So does it mean whether this field was removed by some[ previous PR like this one ](https://github.com/Kong/kong/pull/12343/files#diff-25aed5512e905778adf558ca405a4a15d755d2c8127e4692c05dd6894621df8cR45) ?

Reply:
That is correct. Previously many plugins in CE used different field names to represent the same thing for redis config. We converged on `password` field name for storing redis password and since acme used `auth` for that I needed to adjust it. The `auth` field was removed and replaced by `password` field. However in order to maintain backward compatibility I've added `shorthand_fields` to allow the user to use the plugins with old configurations. 

With these fields the user could still create the plugin using the old names but the admin api responses did not have these fields - you would always get the latest schema in response to your `GET /plugins`, `POST /plugins`, `PATCH /plguins/:plugin_id` etc.. 

This PR adds a functionality to _`translate_backwards`_ shorthand fields which allows admin_api to return responses that include legacy fields by passing a path that describes how to extract the data from a new field under the name of a legacy field.